src/main/java/com/flywithingryd/IngrydAirways/model/Itinerary.java

```diff
@@ -0,0 +1,19 @@
+package com.flywithingryd.IngrydAirways.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity(name = "itinerary")
+public class Itinerary {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private Passenger passenger;
```

Comment:
The passengers too might not be necessary as we can fetch the passengers from the reservation (if you check the reservation class we already have a list of passengers) this skipped my mind.... My apologies

Reply:
Removed